molecule/default/roles/test_glauth/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Pre-install Debian package
+  ansible.builtin.apt:
+    # TODO(dato): can the URL be quoted in two lines, without needing a disable?
+    # yamllint disable-line rule:line-length
+    deb: https://github.com/ASSEKURANSA/glauth/releases/download/debian%2F2.3.2-0_exp5/glauth-binary-dist_2.3.2-0.exp5_amd64.deb
```

Comment:
In that case, I prefer to leave in the clickable URL.

Reply:
well, now you added the `extra-allowed`, so there shouldn't be any complaints. But either way is fine

Reply:
the extra-allowed is only about allowing unnecessary quotes for long strings. But still needs to be split in multiple lines to avoid trigger line-length...

Reply:
yes, but I thought that was what you wanted to do :)